site/content/docs/5.2/getting-started/vite.md

```diff
@@ -0,0 +1,201 @@
+
+```sh
+mkdir {dist,src,src/js,src/scss}
+touch dist/index.html src/js/main.js src/scss/styles.scss src/scss/_custom.scss vite.config.js
+```
+
+When you're done, your complete project should look like this:
+
+```text
+my-project/
+├── dist/
+│   └── index.html
+├── src/
+│   ├── js/
+│   │   └── main.js
+│   └── scss/
+│       ├── _custom.scss
+│       └── styles.scss
+├── package-lock.json
+├── package.json
+└── vite.config.js
+```
+
+At this point, everything is in the right place, but Vite won't work because we haven't filled in our `vite.config.js` yet.
+
+## Configure Vite
+
+With dependencies installed and our project folder ready for us to start coding, we can now configure Vite and run our project locally.
+
+1. **Open `vite.config.js` in your editor.** Since it's blank, we'll need to add some boilerplate config to it so we can start our server. This part of the config tells Vite were to look for our project's JavaScript, where to output the compiled code to (`dist`), and how the development server should behave (pulling from the `dist` folder with hot reload).
+
+   <!-- eslint-skip -->
+   ```js
+   const path = require('path')
+
+   export default {
+     root: path.resolve(__dirname, 'dist'),
+     server: {
+       port: 8080,
+       hot: true
+     }
+   }
+   ```
+
+2. **Next we create our `dist/index.html`.** This is the HTML page Vite will load in the browser to utilize the bundled CSS and JS we'll add to it in later steps. Before we can do that, we have to give it something to render and include the `output` JS from the previous step.
+
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Vite</title>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Vite!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+       <script src="./main.js"></script>
+     </body>
+   </html>
+   ```
+
+   We're including a little bit of Bootstrap styling here with the `div class="container"` and `<button>` so that we see when Bootstrap's CSS is loaded by Vite.
+
+3. **Now we need an npm script to run Vite.** Open `package.json` and add the `start` script shown below (you should already have the test script). We'll use this script to start our local Vite dev server.
+
+   ```json
+   {
+     // ...
+     "scripts": {
+       "start": "vite",
+       "test": "echo \"Error: no test specified\" && exit 1"
+     },
+     // ...
+   }
+   ```
+
+4. **And finally, we can start Vite.** From the `my-project` folder in your terminal, run that newly added npm script:
+
+   ```sh
+   npm start
```

Comment:
At this point atm Vite config is not that good. We can see the following warning in the console: `(!) Could not auto-determine entry point from rollupOptions or html files and there are no explicit optimizeDeps.include patterns. Skipping dependency pre-bundling.`